v3/cypress/fixtures/invalid-docs/index.md

```diff
@@ -0,0 +1,32 @@
+## invalid-content.codap
+This has a invalid content field.
+The invalid content field means that the `isClientContent` function provided to the CFM by CODAP returns false.
+However since there is a content field (even though it is invalid) the CFM thinks this document is "wrapped" so it only sends the value of the content field to CODAP instead of the whole document.
+
+So CODAP receives a document of `{"invalid": "value"}`.
+
+**this does not cause an visible error.**
+It isn't obvious how to handle it better. We don't want to reject documents with unknown fields to allow new fields with crashing old versions of CODAP, so unknown properties are just ignored. We could reject the document because it doesn't have the `type: "CODAP"` or because it doesn't have `content.rowMap` and `content.tileMap` fields, but maybe that is too restrictive?
```

Comment:
```suggestion
It isn't obvious how to handle it better. We don't want to reject documents with unknown fields to allow new fields without crashing old versions of CODAP, so unknown properties are just ignored. We could reject the document because it doesn't have the `type: "CODAP"` or because it doesn't have `content.rowMap` and `content.tileMap` fields, but maybe that is too restrictive?
```